Configuration/Routes/List.yaml

```diff
@@ -0,0 +1,49 @@
+routeEnhancers:
+  ProfileListPlugin:
+    type: Extbase
+    extension: AcademicPersons
+    plugin: List
+    routes:
+      - routePath: '/{page}'
+        _controller: 'Profile::list'
+        _arguments:
+          page: 'demand/currentPage'
+      - routePath: '/{char}'
+        _controller: 'Profile::list'
+        _arguments:
+          char: 'demand/alphabetFilter'
+    defaultController: 'Profile::list'
+    aspects:
+      page:
+        type: StaticRangeMapper
+        start: '1'
+        end: '1000'
+      char:
+        type: StaticValueMapper
```

Comment:
You can also use the StaticRangeMapper for letters. It supports String range so start:'a' and end: 'z' would work.
The rest look really good. 👍